k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java

```diff
@@ -4753,6 +4753,11 @@ private LocalMessage findNewestMessageForNotificationLocked(Context context, Not
      */
     private void notifyAccount(Context context, Account account,
             LocalMessage message, int previousUnreadMessageCount) {
+    	// if it's quiet time and notifications are disabled, then we shouldn't show a notification
```

Comment:
Please remove the comment. If you feel the condition below isn't clear enough you can do something like the following:
```java
boolean isQuietTimeWithNotificationsDisabled = K9.isQuietTime() && K9.getQuietTimeNotificationEnabled();
if (isQuietTimeWithNotificationsDisabled) {
    return;
}
```

Nobody ever corrects comments when the code changes. By avoiding comments we don't have this particular problem and also the code itself should become more readable.